lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced/search-date-range-advanced.component.scss

```diff
@@ -40,6 +40,10 @@
                     margin-top: 2em;
                 }
             }
+
+            .mat-form-field-wrapper:focus-within {
```

Comment:
we should not use material classes, can you use different way?

Reply:
I was looking for the different solution, but it leads to overwriting material. As we use `mat-form-field` and it populates many wrappers. Idk what should we do with it.

Reply:
@MichalKinas @DenysVuika do you have any idea?